### ISSUE TITLE
Error: Cannot find module 'src/server/utils/request'

### ISSUE THREAD

**Issue by mariodian** [apotdevin/thunderhub#482]
I get the following error when running `npm start` after installing and building the project:

````
> thunderhub@0.13.15 start
> cross-env NODE_ENV=production nest start

Error: Cannot find module 'src/server/utils/request'
Require stack:
- /home/satoshi/source/Lightning/thunderhub/dist/modules/security/security.decorators.js
- /home/satoshi/source/Lightning/thunderhub/dist/modules/view/view.controller.js
- /home/satoshi/source/Lightning/thunderhub/dist/modules/view/view.module.js
- /home/satoshi/source/Lightning/thunderhub/dist/app.module.js
- /home/satoshi/source/Lightning/thunderhub/dist/main.js
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:956:15)
    at Function.Module._load (node:internal/modules/cjs/loader:804:27)
    at Module.require (node:internal/modules/cjs/loader:1028:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Object.<anonymous> (/home/satoshi/source/Lightning/thunderhub/src/server/modules/security/security.decorators.ts:7:1)
    at Module._compile (node:internal/modules/cjs/loader:1126:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1180:10)
    at Module.load (node:internal/modules/cjs/loader:1004:32)
    at Function.Module._load (node:internal/modules/cjs/loader:839:12)
    at Module.require (node:internal/modules/cjs/loader:1028:19)
````

**Your environment**
* Version of ThunderHub: 0.13.15
* Deployment method: manual (`npm install && npm run build && npm start`)
* Running lnd 0.15.99-beta, node 16.17.0, npm 8.15.0

**Comment by rednaxus** [comment]
me too

**Comment by brijananand** [comment]
Same issue here when attempting to do a fresh install thunderhub@0.13.15. 

- NPM Version: 8.19.1
- Node Version: 18.9.0,

Do not know how to proceed.

**Comment by apotdevin** [comment]
I can reproduce this. Looking into it

**Comment by apotdevin** [comment]
It should be fixed with the latest commit 383ebeffb782874a76efd8593da624ac69dab50e

Let me know if it works!

**Comment by mariodian** [comment]
Yeah it works thanks! 

Although I ran into other issues (I have to run `BASE_PATH=/thub/ npm run build` in order for BASE_PATH to be included in links such as /home /dashboard etc). Not sure when that changed but it [doesn't seem to be documented](http://docs.thunderhub.io/installation/).
